### ISSUE TITLE
About error bars reported in paper

### ISSUE THREAD

**Issue by kthone** [facebookresearch/DomainBed#103]
Hi. Thanks for the nice work!
I am truly motivated by the work and trying to use it as baseline for my work.
While analyzing the code and the paper, I have a question about the standard deviation reported in the paper. 
According to the paper, std is calculated by running 3 experiments with entirely different random choices.
Can I follow the way you did it by changing hparams_seed, seed, trial_seed in your code?
Thank you!

**Comment by lopezpaz** [comment]
@igul222 Can you shed some light on this one?

**Comment by lopezpaz** [comment]
We try 20 different `hparams_seed` (which selects 20 different hyper-parameter configurations). We repeat each hyper-parameter configuration 3 `trial_seed` times (which re-runs the same choice of hyperparameters to take into account the stochasticity everywhere else, such as data splitting and shuffling). Mean +- stds reported in tables are over the 3 repetitions of the best hyper-parameter configuration.

**Comment by NancyQuris** [comment]
Based on the code, the hparam in each trial is generated by `hash(hparams_seed, trial_seed)`. Each set of hparam is run for only one time. In addition, the result is collected by selecting the best hparm in each trial, while the best hparms among different trials might not be the same. I am afraid that current implementation is not as what you described.

**Comment by lopezpaz** [comment]
You are right, I misremembered. Thanks.

In one sweep, 20 random hyper-parameter configurations (`hparams_seed`) are tried per algorithm. The best hyper-parameter configuration is chosen accordingly to the model selection strategy, and we take note of its test accuracy.

We run 3 of such sweeps (`trial_seed`). We report the mean +- err of the three noted test accuracies.
